### PR TITLE
Fix repl

### PR DESCRIPTION
**Pull request by martinklepsch**
I think this relies on https://github.com/adzerk/boot-cljs/pull/50 but I'm not entirely sure.

### CODE REVIEW

File: src/adzerk/boot_cljs_repl.clj [hunk 1]
```diff
@@ -47,6 +51,18 @@
...
+  "The way Clojurescript handles this does not work when
+   using classloaders in the fancy ways we do."
+  (let [res (pod/classloader-resources "deps.cljs")]
+    (->> (for [[cl uris] res]
+          (if uris
+            (map #(-> % slurp read-string) uris)))
+        flatten
+        distinct
+        (remove nil?)
+        (apply merge-with concat))))
```
Comment (martinklepsch) [inline src/adzerk/boot_cljs_repl.clj]: I'd like some suggestion how to make this more concise if possible.

Comment (alandipert) [inline src/adzerk/boot_cljs_repl.clj]: Maybe

```clojure
(defn get-upstream-deps []
  "The way Clojurescript handles this does not work when
   using classloaders in the fancy ways we do."
  (->> (pod/classloader-resources "deps.cljs")
       (keep second)
       (mapcat identity)
       (map (comp read-string slurp))
       (apply merge-with concat)))
```

I didn't thoroughly test, but `keep` is maybe useful here.

Comment (martinklepsch) [inline src/adzerk/boot_cljs_repl.clj]: Nice didn't know about `keep` thx!

Comment (martinklepsch) [inline src/adzerk/boot_cljs_repl.clj]: @alandipert that seems to work :)

**Comment by martinklepsch** [comment]
@micha can we merge & deploy this please?

**Comment by alandipert** [comment]
Just deployed 0.1.9 also :beers:

**Comment by martinklepsch** [comment]
:beers: :clap:

**Pull request merged**